spec/factories/section.rb

```diff
@@ -2,5 +2,13 @@
   factory :section do
     sequence(:name) { |n| "Section name #{n}" }
     profile
+
+    trait :filled_with_content do
+      after(:create) do |section|
+        4.times do
+          create(:item, Item::TYPES.sample, section: section)
```

Comment:
`sample` add randomness to the spec. I would add one item of each type to cover full testing opportunities in future